### ISSUE TITLE
Does not refresh entries from Trello on chrome

### ISSUE THREAD

**Issue by Hopsakee** [w11k/calendar-for-trello#167]
Calendar for Trello doesn't refresh anymore in Chrome, it does refresh in Firefox. The refresh arrow keeps spinning, but no entries appear in the calendar.

**Comment by fvonberg** [comment]
Hi @Hopsakee 

could you pleas try to "Clear Data". 

1. Open the calendar for trello site
2. Open the context menu on the upper right corner (three horizontal dots, next to "refresh")
3. Select "Clear Data"

Did this fix your problem?

**Comment by Hopsakee** [comment]
Thanks for the reply. I did try that. But it didn't help. 
I fixed the problem by resetting all my chrome settings. Kind of an ugly sollution. But it worked.So for me, this problem is solved. 

    Op donderdag 1 juni 15:11 2017 schreef Frederik von Berg <notifications@github.com> het volgende:
 

 Hi @Hopsakeecould you pleas try to "Clear Data".   
   - Open the calendar for trello site
   - Open the context menu on the upper right corner (three horizontal dots, next to "refresh")
   - Select "Clear Data"
Did this fix your problem?—
